File: GeeksForGeeks/Basic/Starting_point.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int test_case;
    int current_point[2];
    int i, total_paths;
    scanf("%d", &test_case);
    while (test_case--)
    {
        scanf("%d %d", &current_point[0], &current_point[1]);
        scanf("%d", &total_paths);
        int *paths = (int *) malloc(sizeof(int)*2*total_paths);
        for(i = 0; i < total_paths*2; ++i)
        {
            scanf("%d", &paths[i]);
        }
        for(i = 0; i < total_paths*2; i = i + 2)
        {
            current_point[0] -= paths[i];
            current_point[1] -= paths[i+1];
        }
        printf("%d %d\n", current_point[0], current_point[1]);
    }
}

File: GeeksForGeeks/Basic/Starting_point.py
#Python 2.7

if __name__ == '__main__':
    test_case = input()
    while test_case:
        current_point = [int(x) for x in raw_input().split()]
        total_paths = input()
        paths = [int(x) for x in raw_input().split()]
        i = 0
        while i < len(paths):
            current_point[0] -= paths[i]
            current_point[1] -= paths[i+1]
            i += 2
        print current_point[0], current_point[1]
        test_case -= 1
